management/example_test.go

```diff
@@ -1,175 +0,0 @@
-package management_test
```

Comment:
Is this deleted intentionally?

Reply:
> Hi @gagalago, I appreciate the improvements you propose. I'm afraid some of these changes will require changes to the tenant used to run acceptance tests and will likely fail if run.
> 
> I'd be reluctant to merge this PR

in fact, for me, tests in the actual situation failed even on a new tenant. that's why I proposed these changes to be run on a new tenant. I added in the pull request description what/why I did.

What are the changes that will be needed on the tenant?